### DISCUSSION TITLE
component 能传入自定义的组件嘛 我想使用自己封装的date-picker 年范围的选择

### DISCUSSION THREAD

**Discussion by lovesL** [tolking/element-pro-components#465] [Q&A] [answered]
![image](https://github.com/tolking/element-pro-components/assets/27960630/78c79b6b-0229-4f5e-90f0-90b390bea024)

**Comment by InfernalAzazel** [comment, marked as answer]
可以的 作者 写的一个 admin 有个例子就是。
例子: https://github.com/tolking/element-admin-template/blob/main/src/views/demo/Crud.vue
